HotelBooking.API/Controllers/HotelController.cs

```diff
@@ -87,9 +87,102 @@ public async Task<IActionResult> CreateFacilityAsync([FromForm] FacilityRequest
             });
         }
 
+        [Authorize(Roles = "Administrator", AuthenticationSchemes = "Bearer")]
+        [HttpPut("update-facility")]
+        public async Task<IActionResult> UpdateFacilityAsync(FacilityModel model, Guid id)
+        {
+            if (model == null)
+            {
+                logger.LogError("Invalid model sent from client.");
+                return BadRequest("Invalid model object");
```

Comment:
I think we don't care about invalid data before triggering business logic. So this error could be redundant for me

Reply:
I fixed it. thanks